### PR TITLE
Feature/bracket on separate line

### PR DESCRIPTION
**Pull request by nojaf**
This is the first attempt for #453.

### CODE REVIEW

File: src/Fantomas/FormatConfig.fs [hunk 1]
```diff
@@ -39,6 +40,7 @@ type FormatConfig =
           ReorderOpenDeclaration = false
           SpaceAroundDelimiter = true
           KeepNewlineAfter = false
+          BracketOnSeparateLine = false // TODO: add to cli
```
Comment (jindraivanek) [inline src/Fantomas/FormatConfig.fs]: TODO?

Comment (nojaf) [reply src/Fantomas/FormatConfig.fs]: That has been done, the comment needs to removed.

**Comment by fahrenq** [comment]
Hey guys,

Any estimate on this?

Could you maybe use any help from me? I'm ready to take over if needed, just please provide a little bit of context :)

**Comment by nojaf** [comment]
Hello @fahrenq, I'm picking this up early 2020 in collaboration with the team at G-Research.
I cannot give any estimate I'm afraid but it will have the highest priority in the upcoming sponsorship contract.

**Comment by nojaf** [comment]
Hello @fahrenq, the internal conversations about this are still ongoing. It will take some time before I can start working on this again. Just so you know.

**Comment by fahrenq** [comment]
Hey Florian,

Thank you very much for the update. Appreciate it!

**Comment by nojaf** [comment]
Hey @fahrenq , I messed this branch and by accident closed this PR.
The intention to work on this is still there, will open a new PR when I have something.

**Comment by fahrenq** [comment]
Awesome! Thank you :)

**Pull request closed without merging**